### PR TITLE
Add dnslow.me DNS in DNS Providers

### PR DESCRIPTION
**Pull request by PeterDaveHello**
It's my side project, please feel free to give it a try!

### CODE REVIEW

File: docs/general/dns-providers.md [hunk 1]
```diff
@@ -1125,3 +1125,19 @@ Blocks more ads and malware than AdGuard DNS thanks to more advanced syntax, but
 | DNS, IPv4      | Varies; see link above. |
 | DNS, IPv6      | Varies; see link above. |
 | DNSCrypt, IPv4 | Varies; see link above. |
+
+#### dnslow.me
+
+[dnslow.me](https://dnslow.me/) is an advertisement and threat blocking, privacy-first, encrypted DNS, also a free and open source project on GitHub.
```
Comment (ss-orlova) [inline docs/general/dns-providers.md:1131]: I suggest dividing this sentence in multiple parts.

`[dnslow.me](https://dnslow.me/) is a free privacy-first DNS service that blocks ads and protects you from threats. Its source code is available on GitHub.`

File: docs/general/dns-providers.md [hunk 2]
```diff
@@ -1125,3 +1125,19 @@ Blocks more ads and malware than AdGuard DNS thanks to more advanced syntax, but
...
+
+Only encrypted DNS protocols are provided, plain-text DNS is not supported here.
+
+All DNS requests will be protected with threat-intelligence feeds and randomly distributed to some other DNS resolvers for enhanced privacy.
+
+Privacy policy: Logging is only enabled to debug, and improve the service itself, minimize the false-positive blocking. All logs will only be existing for a very short time. No logs will be shared, sold, or exchanged with any 3rd-party.
```
Comment (ss-orlova) [inline docs/general/dns-providers.md:1137]: Maybe also be specific about what a short period of time means?

or

`Privacy policy: logging is only used to debug, improve the service, and minimize false-positive blocking. All logs are stored for a short period of time only. Logs aren't shared, sold, or exchanged with any third party.`

**Comment by el-termikael** [comment]
@PeterDaveHello hello,
Could you kindly make changes suggested by @ss-orlova?

**Comment by PeterDaveHello** [comment]
Hi @el-termikael @ss-orlova!

I put the config here: https://github.com/PeterDaveHello/dnslow.me/blob/master/adguardhome/conf/AdGuardHome.yaml, but still trying to get the balance of debugging and keep the least of logs, might do some manual change during digging, could be hard if I write it here, and need to change the wording everytime it's changed. Or maybe you'd suggest me to just remove that on the description? Thanks!

**Comment by el-termikael** [comment]
@PeterDaveHello Questions remain so I have to close the issue. Please reopen it if you come up with the wording and make changes requested by @ss-orlova

**Pull request closed without merging**